providers/base/bin/pipewire_utils.py

```diff
@@ -68,6 +69,25 @@ class PipewireTest:
     """
     logger = logging.getLogger()
 
+    def _run_command(self, cmd: dict, shell=False) -> str:
+        """
+        use subprocess.check_output to execute command
+
+        :param cmd: the command will be executed
+
+        :param shell: enbale shell or not
+
+        :returns: Ouput of command
+        """
+        try:
+            return subprocess.check_output(cmd,
+                                           shell=shell,
+                                           universal_newlines=True)
+
+        except (subprocess.CalledProcessError, FileNotFoundError) as e:
+            raise SystemExit("Running command:{} failed due to {}"
+                             .format(cmd, repr(e)))
+
```

Comment:
This won't be necessary if we just add the compare_status function to pipewire_utils.py